examples/fscloud/README.md

```diff
@@ -1,6 +1,6 @@
 # Financial Services Cloud profile example
 
-An end-to-end example that uses the [Profile for IBM Cloud Framework for Financial Services](../../profiles/fscloud/) to deploy a VSI.
+An end-to-end example that uses the [Profile for IBM Cloud Framework for Financial Services](../../modules/fscloud/) to deploy a VSI.
```

Comment:
`../../modules/fscloud/` link won't work in hashicorp registry -> https://registry.terraform.io/modules/terraform-ibm-modules/landing-zone-vsi/ibm/latest/examples/fscloud
Please update to `https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/tree/main/modules/fscloud`